File: app/src/main/java/com/example/MainActivity.java
package com.example;

import android.app.Activity;
import android.content.Intent;
import android.os.Bundle;

import androidx.annotation.Nullable;

import com.example.fishyu.fishdemo.R;
import com.example.updater.ApplicationUpdater;
import com.example.updater.InstallApkCompat;

public class MainActivity extends Activity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

//        ApplicationUpdater updater = new ApplicationUpdater(this);
//        updater.checkNewVersion();
    }


    @Override
    protected void onActivityResult(int requestCode, int resultCode, Intent data) {
        super.onActivityResult(requestCode, resultCode, data);
//        InstallApkCompat.onActivityResult(this, requestCode, resultCode, data);
    }
}

File: app/src/main/java/com/demo/fishdemoxhb/MainActivity.kt
package com.demo.fishdemoxhb

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import com.demo.fishdemoxhb.test.DemoActivity

class MainActivity : AppCompatActivity(), View.OnClickListener {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

    }

    override fun onClick(p0: View?) {
        startActivity(Intent(this, DemoActivity::class.java))
    }
}